packages/core/src/types.ts

```diff
@@ -16,6 +16,16 @@ export type AppMetadata = Omit<AppJsonMetadata, "image"> & {
   image: string | MetaplexFile;
 };
 
+export type MetaplexFileReleaseJsonMetadata = Omit<ReleaseJsonMetadata, "image"> & {
+  image: string | MetaplexFile;
```

Comment:
If `media` and `files` can be strictly `MetaplexFile` objects, can we do the same for `image`, to make the modifications to the generated type consistent?

Also, why is `Omit` needed for `image`, but not for `extensions.solana_dapp_store.[media|files]`?

Reply:
I'm wanting to move into a world where our schema files are the source of truth for this stuff; can I use advanced types in schemas?

As for why Omit is used...oddly...I tried, well, omitting using it, and things appeared to work. Might not be needed.

Reply:
Update: I cannot build without `Omit`-ing "image". But for some reason the same run isn't required for `media|files`. Unfortunately right now I don't know why...

Reply:
Alright, I can approve as-is, but please file an issue to get this type cleaned up. This has some code smell about it...

Reply:
Okay yeah this all centers around the tension between what the Metaplex sdk needs and our despire to validate the JSON against the schema.

You're totally right it smells. I'll get it addressed along side some of the other schema cleanup I'm looking into.